docs/source/getting-started/examples/histograms.ipynb

```diff
@@ -83,34 +83,42 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
+   "execution_count": 2,
    "metadata": {
     "pycharm": {
      "name": "#%%\n"
     }
    },
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/var/folders/lc/l1nrfls97n591dg9pb5dybzc0000gq/T/ipykernel_20412/3114099242.py:5: DeprecationWarning: Call to deprecated function (or staticmethod) make_split_dataframe. (Use Polars instead) -- Deprecated since version 0.12.0.\n",
+      "  dp.t.make_split_dataframe(separator=\",\", col_names=col_names) >>\n",
+      "/var/folders/lc/l1nrfls97n591dg9pb5dybzc0000gq/T/ipykernel_20412/3114099242.py:6: DeprecationWarning: Call to deprecated function (or staticmethod) make_select_column. (Use Polars instead) -- Deprecated since version 0.12.0.\n",
+      "  dp.t.make_select_column(key=\"educ\", TOA=str) >>\n"
+     ]
```

Comment:
Guess we just haven't rerun the notebook since then? Not the end of the world, but would like to clean that up. (Though it doesn't necessarily need to block this PR.)

Reply:
I don't like this notebook at all. How about we remove it completely?

Reply:
It's very outdated, we have better ways of doing this in Polars now, and we have much more thorough documentation for the thresholded mechanism if someone wants to be low-level and use the framework APIs directly.